paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AutocompleteActivity.kt

```diff
@@ -39,7 +42,12 @@ internal class AutocompleteActivity : AppCompatActivity() {
         }
 
         WindowCompat.setDecorFitsSystemWindows(window, false)
-        starterArgs.appearance.parseAppearance()
+
+        val appearanceContext = starterArgs.appearanceContext
+
+        if (appearanceContext is AutocompleteAppearanceContext.PaymentElement) {
+            appearanceContext.appearance.parseAppearance()
+        }
```

Comment:
Nit: We could add a `applyAppearance()` on `AutocompleteAppearanceContext` to avoid logic at the call-site, and then add a comment why it's not required to be applied in Link.